DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: spimalot
 Title: Support for 'sircovid'
-Version: 0.6.12
+Version: 0.6.13
```

Comment:
Do you need your changes from sircovid here? If so then increase the required version number below. If not then we will merge this before the sircovid PR

Reply:
Yes they are needed here as the size of the output changes and spimalot expects a new length of state trajectories produced from sircovid. Have bumped the version up now thank you.